src/Controls/tests/TestCases.HostApp/Issues/Issue25912.xaml

```diff
@@ -0,0 +1,16 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<controls:TestContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+			 xmlns:controls="clr-namespace:Maui.Controls.Sample.Issues"
+             x:Class="Maui.Controls.Sample.Issues.Issue25912MainPage">
+
+			 <ContentPage.ToolbarItems>
+				<ToolbarItem x:Name="toolbarItem" IconImageSource="{FontImage Glyph='+',Color=Red,Size=30}"/>
```

Comment:
Could include a secondary toolbar item?

Reply:
When using FontIconSource in a secondary toolbar, the view appears improper on iOS and Mac platforms. This behavior is also noted in the official documentation:
[Primary and Secondary Toolbar Items Documentation](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/toolbaritem?view=net-maui-9.0#primary-and-secondary-toolbar-items)